tests/custom_cmake_build/xc_override.cmake

```diff
@@ -0,0 +1,4 @@
+list(FIND CMAKE_C_SOURCE_FILE_EXTENSIONS xc XC_FOUND)
```

Comment:
Why do you need a local copy of this file when you are fetching it from xmos_cmake_toolchain in the Jenkinsfile?
(Also `tests/custom_cmake_build/xs3a.cmake`)

Reply:
Accidental commit by the looks of things.. Removed